common/app/mvt/MultiVariateTesting.scala

```diff
@@ -47,6 +47,17 @@ object ABNewHeaderVariant extends TestDefinition(
   }
 }
 
+object ABGalleryRedesignVariant extends TestDefinition(
+  variants = Nil,
+  name = "ab-gallery-redesign-variant",
+  description = "Feature switch (0% test) for gallery redesign",
+  sellByDate = new LocalDate(2016, 5, 10)
+) {
+  override def isParticipating(implicit request: RequestHeader): Boolean = {
+    request.headers.get("X-GU-ab-gallery-redesign").contains("variant") && switch.isSwitchedOn && ServerSideTests.isSwitchedOn
```

Comment:
Thanks for asking @desbo. As far as I am aware, if we want to run a serverside test on 30% of people (made up number) we need a new header, and to override `isParticipating`